ebu_tt_live/node/base.py

```diff
@@ -82,6 +83,13 @@ def is_document(self, document):
         else:
             return False
 
+    def has_document_sequence_identifier_used(self, document):
```

Comment:
I'm having trouble understanding what this function might do from its name. Maybe something like `limit_sequence_to_one` would be better? This is so that it is obvious that we should only call it when we actually want to limit the number of input sequences. Happy for other ideas, but it definitely needs to change.